.storybook/webpack.config.js

```diff
@@ -2,14 +2,7 @@ const path = require('path');
 
 module.exports = ({config}) => {
     config.module.rules.push({
-        test: /\.scss$/,
-        loaders: ["style-loader", "css-loader", "sass-loader"],
-        include: path.resolve(__dirname, '../src')
-    });
-    config.resolve.extensions.push('.scss');
-
-    config.module.rules.push({
-        test: /\.(ts|tsx)$/,
+        test: /(?:test)\.(ts|tsx)$/,
```

Comment:
If the described issue is fixed, I think this can be removed.

Reply:
Turns out that this was not the case. There was still no Typescript output from the story files when using the old configuration.
We don't need the files containing `.test` in Storybook anyway though so we don't have to find a solution around this.